src/home/scraper/issue-scraper.ts

```diff
@@ -0,0 +1,49 @@
+import { checkSupabaseSession } from "../rendering/render-github-login-button";
+
+export async function startIssueScraper(username: string) {
+  const supabaseAuth = await checkSupabaseSession();
+
+  // Get the last fetch timestamp
+  const lastFetchKey = `lastFetch_${username}`;
+  const lastFetch = localStorage.getItem(lastFetchKey);
+  const now = Date.now();
+
+  // Prepare the request payload
+  const requestBody: { authToken: string; timestamp?: number } = {
+    authToken: supabaseAuth.provider_token,
+  };
+
+  if (lastFetch) {
+    const lastFetchTimestamp = Number(lastFetch);
+    if (now - lastFetchTimestamp < 24 * 60 * 60 * 1000) {
```

Comment:
I wonder if there is a better way to implement this. For example, instead of by timing, we can do based on an event. 

Although I can't think of how it would be possible in outside organization contexts.